charts/gitlab-monitor/templates/ingress.yaml

```diff
@@ -7,11 +7,15 @@ metadata:
     chart: "{{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}"
 spec:
   rules:
-  - host: {{ .Release.Name }}.{{ .Values.basedomain }}
+  - host: {{ .Values.ingress.domain }}
```

Comment:
I'm not familiar with Helm. This doesn't break existing charts, does it?

Reply:
Yes
Helm charts are locked with the version number in `charts/gitlab-monitor/Chart.yaml`.
Unless people upgrade the chart version, there is no impact.